### PR TITLE
Add video v1p1beta samples for face detection and video transcription

### PR DESCRIPTION
**Pull request by nnegrey**

### CODE REVIEW

File: video/beta/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+```
+export GOOGLE_APPLICATION_CREDENTIALS=/path/to/your-project-credentials.json
+```
+
+After you have authorized, you can analyze videos.
+
+Detect Faces' Bounding Boxes
+```
+java -cp target/video-google-cloud-samples-beta-1.0.8-jar-with-dependencies.jar \
```
Comment (dzlier-gcp) [inline video/beta/README.md]: Generally we prefer to use `mvn exec:java` for command line instructions over running the jar. See the [auth sample](https://github.com/GoogleCloudPlatform/java-docs-samples/tree/master/auth#listing-buckets-with-default-credentials) for an example.

Comment (nnegrey) [reply video/beta/README.md]: Perfect. I totally agree!

File: video/cloud-client/src/main/java/com/example/video/QuickstartSample.java [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2017 Google Inc.
+ * Copyright 2018 Google Inc.
```
[the remark this answers is not in the document]
Comment (nnegrey) [reply video/cloud-client/src/main/java/com/example/video/QuickstartSample.java]: Fixed

File: video/cloud-client/src/test/java/com/example/video/QuickstartIT.java [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2017 Google Inc.
+ * Copyright 2018 Google Inc.
```
[the remark this answers is not in the document]
Comment (nnegrey) [reply video/cloud-client/src/test/java/com/example/video/QuickstartIT.java]: Fixed

File: video/cloud-client/src/test/java/com/example/video/DetectIT.java [hunk 4]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2017 Google Inc.
+ * Copyright 2018 Google Inc.
```
[the remark this answers is not in the document]
Comment (nnegrey) [reply video/cloud-client/src/test/java/com/example/video/DetectIT.java]: Fixed

File: video/cloud-client/README.md [hunk 5]
```diff
@@ -36,38 +36,32 @@ export GOOGLE_APPLICATION_CREDENTIALS=/path/to/your-project-credentials.json
...
-Detect Faces
-```
-java -cp target/video-google-cloud-samples-1.0.0-jar-with-dependencies.jar \
-    com.example.video.Detect faces gs://demomaker/google_gmail.mp4
-```
-
 Detect Labels
 ```
-java -cp target/video-google-cloud-samples-1.0.0-jar-with-dependencies.jar \
+java -cp target/video-google-cloud-samples-1.0.8-jar-with-dependencies.jar \
```
[the remark this answers is not in the document]
Comment (kurtisvg) [reply video/cloud-client/README.md]: I think you converting these.

**Pull request merged**